### PR TITLE
PRIPAD-5194 Tidy up HTML out of PHP

### PR DESCRIPTION
**Pull request by sebskuse**
In the current version of HockeyKit it has the various pages that render displaying an app all jammed in the same index.php inbetween a load of `if`'s. 

I've modified this to use a `view` class, and a `renderer` class to render the page that's been requested, and all the HTML now lives in a `views` directory. The views contain placeholders that are replaced at runtime from within the `renderer` which should help with maintainability.

### CODE REVIEW

File: server/php/includes/renderer.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        $this->_superview = new view("superview.html");
+        $this->_content = new view();
+    }
+    
+    public function setDevice($device) {
+        $applications = array_filter($this->_appUpdater->applications, function ($item) use ($device) {
+            switch($device) {
+                case Device::iOS:
+                    return $item[AppUpdater::INDEX_PLATFORM] == AppUpdater::APP_PLATFORM_IOS;
+                    break;
```
Comment (dhardiman) [inline server/php/includes/renderer.php]: `return` and `break`? Is this a phpism or can it go?

**Comment by dhardiman** [comment]
As far as my knowledge of the subject goes, looks fine to me.

File: server/php/includes/renderer.php (hunk 1 above)
Comment (sebskuse) [inline server/php/includes/renderer.php]: That can almost certainly go actually

**Comment by sebskuse** [comment]
Updated

**Comment by sebskuse** [comment]
Bump

**Pull request merged**